WordPress/JetpackStatsWidgets/LockScreenWidgets/LockScreenStatsWidget.swift

```diff
@@ -33,5 +33,6 @@ struct LockScreenStatsWidget<T: LockScreenStatsWidgetConfig>: Widget {
         .configurationDisplayName(config.displayName)
         .description(config.description)
         .supportedFamilies(config.supportFamilies)
+        .iOS17ContentMarginsDisabled() /// Temporarily disable additional iOS17 margins for widgets
```

Comment:
**Super nitpick**, `///` is a documentation comment, as opposed to the standard `//`. With `///` Xcode and other tools will render the comment as documentation reading tags and markdown. However, the comment needs to be on a type or method declaration. We don't need to "document" method calls. I just wanted to point it out but I don't expect this to be addressed. 👍 😄